### DISCUSSION TITLE
Select onchange submit form

### DISCUSSION THREAD

**Discussion by markdynamo6** [filamentphp/filament#8340] [Help] [answered]
I'm using the Select component. How do I trigger a livewire:change to submit the form when the value of the select changes?

**Comment by leandrocfe** [comment]
```php
->extraInputAttributes(['wire:change' => 'create']) //'save' to EditPage
```

**Reply by ahmadrio** [reply]
thanks it works in custom page

**Comment by markdynamo6** [comment]
Yep. After rummaging around in the source, I found that one as well. Cheers.

**Comment by danharrin** [comment, marked as answer]
I would use afterStateUpdated() instead

**Reply by markdynamo6** [reply]
Any particular reason?

**Reply by danharrin** [reply]
Because `extraInputAttributes()` should be a last resort, its leaking HTML into a PHP class, you get autocomplete when calling a PHP method from PHP, `extraInputAttributes()` only exists for a subset of fields

**Reply by markdynamo6** [reply]
Cheers
